Plutarch.hs

```diff
@@ -22,9 +23,8 @@ module Plutarch (
   PI.punsafeConstant,
   PI.Term,
   PI.TermCont (..),
-  PL.plift,
```

Comment:
It looks like `plift` and `pconstant` exports have been removed from the `Plutarch` module. Should probably be added back.

Reply:
I'm pretty sure this was a mistake in the first place, since it makes no sense. They should be re-exported from `Plutarch.Prelude`, not `Plutarch`.

Reply:
Sounds good as well. Maybe this should be tackled alongside the whole "add more exports to prelude" thing. Related: #28